docs/snippets/react/button-story-rename-story.js.mdx

```diff
@@ -5,7 +5,13 @@ import React from 'react';
 
 import { Button } from './Button';
 
-export const Primary = () => <Button primary label="Button"/>;
+export default {
+  component: Button,
+};
+
+export const Primary = {
+  render: () => <Button primary label="Button" />,
+};
```

Comment:
I love that you kept these examples instead of going pure object args ❤️

Reply:
Indeed, based on the documentation I thought about going with pure object args. But in cases such as that one, it's better if we included the render function to help out our readers.